tests/simple-node-nixops.nix

```diff
@@ -0,0 +1,26 @@
+{ pkgs, ... }:
+let
+  iohk-nixops = pkgs.fetchFromGitHub {
+    owner = "input-output-hk";
+    repo = "iohk-nixops";
+    rev = "1777ae1edcfd64696e72e35de9c4b3bc947ba43e";
+    sha256 = "0xy6m1lgrwbn1vnnzgrn7fpccvfzf9l3fnrzc11bpvcag6rkyzbd";
+  };
+in {
+  name = "boot";
+  nodes = {
+    machine = { config, pkgs, ... }: {
+      imports = [ (import (iohk-nixops + "/modules/cardano-node-config.nix") 0 "") <nixops/nix/options.nix> <nixops/nix/resource.nix> ];
+      services.cardano-node = {
+        autoStart = true;
+        initialPeers = [];
+      };
+    };
+  };
+  testScript = ''
+    startAll
+    $machine->waitForUnit("cardano-node.service");
+    # TODO, implement sd_notify?
```

Comment:
FYI: code now calls Systemd.notifyReady to get rid of this, although it should be a separatge issue outside of this PR.

Reply:
oops, it is using notify now, i'll fix that